app/views/users/index.html.erb

```diff
@@ -1,164 +1,101 @@
-      quis auctor auctor, turpis sem eleifend dui, et consequat nisl nisi tincidunt metus.</p>
-    <p>Fusce bibendum in ex quis venenatis. Phasellus tincidunt massa at nulla porttitor, ac mattis odio luctus. Proin
-      cursus, nulla quis auctor auctor, turpis sem eleifend dui, et consequat nisl nisi tincidunt metus.Fusce bibendum
-      in ex quis venenatis. Phasellus tincidunt massa at nulla porttitor, ac mattis odio luctus. Proin cursus, nulla
-      quis auctor auctor, turpis sem eleifend dui, et consequat nisl nisi tincidunt metus.</p>
-    <p>Fusce bibendum in ex quis venenatis. Phasellus tincidunt massa at nulla porttitor, ac mattis odio luctus. Proin
-      cursus, nulla quis auctor auctor, turpis sem eleifend dui, et consequat nisl nisi tincidunt metus.Fusce bibendum
-      in ex quis venenatis. Phasellus tincidunt massa at nulla porttitor, ac mattis odio luctus. Proin cursus, nulla
-      quis auctor auctor, turpis sem eleifend dui, et consequat nisl nisi tincidunt metus.</p>
-    <p>Fusce bibendum in ex quis venenatis. Phasellus tincidunt massa at nulla porttitor, ac mattis odio luctus. Proin
-      cursus, nulla quis auctor auctor, turpis sem eleifend dui, et consequat nisl nisi tincidunt metus.Fusce bibendum
-      in ex quis venenatis. Phasellus tincidunt massa at nulla porttitor, ac mattis odio luctus. Proin cursus, nulla
-      quis auctor auctor, turpis sem eleifend dui, et consequat nisl nisi tincidunt metus.</p>
-
-  </div>
-</section>
-
-
-<footer>
-  <p> Copyright &copy; the best team </p>
-</footer>
-<!-- New -->
-<!-- My driver -->
-<!-- Shared with me --> <!-- Sorted by date-->
-<!-- Recent --> <!-- File which were opened recently -->
-<!-- Starred -->
-<!-- _________ -->
-<!-- Storage --> <!-- Not clickable, showing just how much storage is left-->
-
-<!-- My Drive with the options of creating folders etc. -->
-<!-- List view/ Grid view-->
-<!-- Details about the recent activity or details about the current drive or the current folder-->
-<!-- _________________ -->
-<!-- Folders -->
-<!-- Files -->
-<!-- Organized -->
-
 
+<body>
+	<div class="container">
+		<header>
+			<div class="toolbar" role="banner">
+			  <img width="40" alt="Logo" src="images/logo.png" class="header-image">
+  				<span>Universal Storage Tool</span>
+    				<div class="spacer"></div>
+				<div class="topnav">
+					<label>
+						<input type="text" placeholder="Search..">
+					</label>
+				</div>
+    			<img src="https://picsum.photos/50" alt="account info" class="header-image">
+			</div>
+
+<!--			TO DO: Make the logo go to home page-->
+<!--					Align the search-bar with the main-->
+<!--					-->
+		</header>
+
+		<main> <!-- THIS WILL BE THE MAIN PART OF THE PAGE -->
+			<section>
+				<!-- THE MAIN FILES SECTIONS -->
+				 <hr>
+	        	<div class="item-container">
+<!--					HERE WILL BE THE FILES-->
+							<% @session.files.each do |file| %>
```

Comment:
Currently the files and their buttons are overlapping, like you have a file, the buttons, and another file title on the same row. I believe they should be put in containers but I figure you got that covered. Just made sure to point it out ^^